configure.ac

```diff
@@ -6,7 +6,7 @@
 # Use and distribution licensed under the BSD license.  See
 # the COPYING file in this directory for full text.
 
-AC_INIT([libdrizzle],[5.1.4],[http://bugs.launchpad.net/drizzle],[libdrizzle],[http://drizzle.org/])
```

Comment:
AFAICS it is simply not possible to define version number dynamically, cf. section for **AM_INIT_AUTOMAKE** https://www.gnu.org/software/automake/manual/html_node/Public-Macros.html